config/routes.rb

```diff
@@ -13,5 +13,10 @@
 
   resources :orders
 
+  namespace :admin do
+    resources :users, only: [:show, :index]
+     get "/dashboard", to: "users#show"
```

Comment:
I think we don't need the :show on line 17 because we are setting it explicitly on the line 18.